accessible/field-segment.component.js

```diff
@@ -144,7 +150,12 @@ blocklyApp.FieldSegmentComponent = ng.core.Component({
   // Confirm a selection for dropdown fields.
   selectOption: function() {
     if (this.optionValue == Blockly.Msg.RENAME_VARIABLE) {
```

Comment:
Yep, we've been talking about how to fix this for a bit. Neil's change to update all the fields to use setText/setValue correctly is a start as it lets us use different text and values. We've also been discussing prefixing all variable values so we can do a value comparison, but a text comparison is problematic for translation and reserved words. Short term might be having the dropdown maintain which indices those special values are in and ask each time.